deep_quoridor/src/agents/alphazero/nn_evaluator.py

```diff
@@ -34,6 +34,8 @@ def evaluate(self, game: Quoridor):
         # Rotate the board if player 2 is playing so that we always work with player 1's perspective.
         game, is_board_rotated = self.rotate_if_needed_to_point_downwards(game)
 
+        self.network.eval()  # Disables dropout
```

Comment:
Just to be 100% sure, we *always* want to disable dropout for evaluation, right? 
I mean, I see how we want to do this for testing in order to achieve repeatability. But during actual play / execution ... ? Isn't the point of the dropout to generate some randomness?
I suppose we would only use that during training, as regularization, but during execution as a "currently frozen" policy it should be OK to remove the Dropout, right?
Just want to make sure before this ends up hardcoded here (instead of with a "is_testing" flag). 
I think it's OK as it is, just wanted to suggest making sure of it.